core/domain/learner_playlist_services_test.py

```diff
@@ -79,9 +80,6 @@ def setUp(self):
             category='Welcome')
 
     def _get_all_learner_playlist_exp_ids(self, user_id):
-        """Returns the list of all the exploration ids in the learner's playlist
```

Comment:
Don't remove this docstring (this is what is failing the travis build)

Reply:
Done